types/address.go

```diff
@@ -386,6 +390,62 @@ func (ca ConsAddress) Format(s fmt.State, verb rune) {
 	}
 }
 
+// SmartChainAddress defines a standard smart chain address
+type SmartChainAddress [SmartChainAddressLength]byte
+
+// NewSmartChainAddress is a constructor function for SmartChainAddress
+func NewSmartChainAddress(addr string) (SmartChainAddress, error) {
```

Comment:
I think we dont have to implement this function, because we get `[]byte` and `SetBytes` will be good enough.

Reply:
What I mean is we can simplify the logic:
```	
delBscAddr := hex.EncodeToString(delBscAddrAcc.Bytes())
recipient, err := sdk.NewSmartChainAddress(delBscAddr
```
into 
```
var recipient = new(sdk.SmartChainAddress)
recipient.SetBytes(delBscAddrAcc.Bytes())
```